ee/tables/dataflattentable/tables.go

```diff
@@ -137,8 +138,15 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 	var results []map[string]string
 
 	requestedPaths := tablehelpers.GetConstraints(queryContext, "path")
-	if len(requestedPaths) == 0 {
-		return results, fmt.Errorf("The %s table requires that you specify a single constraint for path", t.tableName)
+	requestedRawDatas := tablehelpers.GetConstraints(queryContext, "raw_data")
+
+	if len(requestedPaths) == 0 && len(requestedRawDatas) == 0 {
+		return results, fmt.Errorf("The %s table requires that you specify at least one of 'path' or 'raw_data'", t.tableName)
+	}
+
+	flattenOpts := []dataflatten.FlattenOpts{
+		dataflatten.WithSlogger(t.slogger),
+		dataflatten.WithNestedPlist(),
```

Comment:
Why this plist-related option?

Reply:
This code handles plist in addition to json, xml, ini, jwt. I just copied these options from the original [generatePath](https://github.com/kolide/launcher/blob/main/ee/tables/dataflattentable/tables.go#L174) function to consolidate them here for both `generatePath` and `generateRawData`

Reply:
`dataflatten.WithNestedPlist`is an older attempt at handling this nesting. We should carry it through